src/mp3.c

```diff
@@ -227,16 +241,72 @@ mp3_read_2d	(SF_PRIVATE * psf, double * ptr, sf_count_t len)
 #define MAX_SAMPLES_ENCODED 1024
 #define ENC_BUFFER_SIZE (1.25 * MAX_SAMPLES_ENCODED) + 7200
 
+static LAME_LIB_HANDLE lame_handle = {} ;
+
+static char *
+mp3_init_lame ()
+{	int failed = 0 ;
+	char * err = NULL ;
+	while (lame_handle.dlhandle == NULL)
+	{	lame_handle.dlhandle = dlopen ("liblame.so", RTLD_LAZY) ;
+		if (lame_handle.dlhandle == NULL)
+			return dlerror () ;
+		lame_handle.init = dlsym (lame_handle.dlhandle, "lame_init") ;
+		if (lame_handle.init == NULL)
+		{	failed = 1 ;
+			break ;
+		}
```

Comment:
Those braces don't match the coding style of the rest of the libsndfile code base,

The `Scripts/cstyle.py` script catches some, but not all issue.

Reply:
I've just added some new rules to the `Script/cstyle.py` script that should help with some more coding style issues. The one that's difficult to add to that script is the closing curly indentation which the current source code uses. It should be like:
```
    if (whatever)
    {   do_soometing ;
        } ;
```
Notice how the closing curly is indented one more level than the open curly brace.